.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Bug
+about: "Report confirmed bugs. For unconfirmed bugs please visit https://discuss.elastic.co/c/beats"
+
+---
+
+Please post all questions and issues on https://discuss.elastic.co/c/beats
+before opening a Github Issue. Your questions will reach a wider audience there,
+and if we confirm that there is a bug, then you can open a new issue.
+
+For security vulnerabilities please only send reports to security@elastic.co.
+See https://www.elastic.co/community/security for more information.
+
+For confirmed bugs, please report:
+- Version: 
+- Operating System: 
+- Discuss Forum URL:
+- Steps to Reproduce: 
```

Comment:
Maybe note that adding the configuration used is also helpful.

Reply:
@ph see line 14 :)